t/admin/upstream-array-nodes.t

```diff
@@ -450,3 +450,38 @@ GET /t
 {"error_msg":"invalid configuration: property \"nodes\" validation failed: object matches none of the required"}
 --- no_error_log
 [error]
+
+
+
+=== TEST 14: nodes host include ipv6 addr
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "upstream": {
+                        "nodes": [
+                            {
+                                "host":"[::1]",
```

Comment:
Is it valid only when using object's node? What happens if I use an array pattern like `[::1]:8080`?

Reply:
It will not affect object's node, see here: https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L308